inspectit-oce-core/src/test/java/rocks/inspectit/oce/core/instrumentation/config/DataProviderResolverUnitTest.java

```diff
@@ -0,0 +1,197 @@
+package rocks.inspectit.oce.core.instrumentation.config;
+
+import org.assertj.core.util.Maps;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import rocks.inspectit.oce.core.config.model.instrumentation.InstrumentationSettings;
+import rocks.inspectit.oce.core.config.model.instrumentation.dataproviders.GenericDataProviderSettings;
+import rocks.inspectit.oce.core.instrumentation.config.model.ResolvedGenericDataProviderConfig;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DataProviderResolverUnitTest {
+
+    @InjectMocks
+    private DataProviderResolver resolver = new DataProviderResolver();
+
+    @Nested
+    public class ResolveDataProviders {
+
+        private static final String PROVIDER_NAME = "My-Provider1";
+        private static final String PROVIDER_VALUE_BODY = "return new Integer(42);";
+
+        InstrumentationSettings config;
+        GenericDataProviderSettings inputProvider;
+
+        @BeforeEach
+        void setup() {
+            config = new InstrumentationSettings();
+            inputProvider = new GenericDataProviderSettings();
+            inputProvider.setValueBody(PROVIDER_VALUE_BODY);
+            config.setDataProviders(Maps.newHashMap(PROVIDER_NAME, inputProvider));
+        }
+
+        @Test
+        void verifyCaseSensitivity() {
```

Comment:
I think I don't get the name of this test. Are you testing some casing stuff in this test? I mean, you basically just check if the name is correctly resolved, or do I miss something?

Reply:
I think I can jsut remove this test. This was a left over when I had the stuff case insensitive